DSharpPlus/Enums/Permission.cs

```diff
@@ -268,6 +268,12 @@ public enum Permissions : long
         [PermissionString("Manage emoji")]
         ManageEmojis = 0x0000000040000000,
 
+        /// <summary>
+        /// Allows managing guild stickers.
+        /// </summary>
+        [PermissionString("Manage stickers")]
```

Comment:
This isn't an API exposed permission.

Reply:
No, but I was advised against renaming ManageEmojis to ManageEmojisAndStickers as it is in the API because it'd be considered a breaking change.

Reply:
Well I don't see sticker as a word on the permissions page in the docs.

Reply:
If ManageEmojis is the permission handling Emojis _and_ stickers, I'd recommend instead just adding it to the comment to not break compatibility and/or API consistency (for whatever API consistency we may have).